src/source/Signaling/Signaling.c

```diff
@@ -961,6 +983,19 @@ STATUS getChannelEndpoint(PSignalingClient pSignalingClient, UINT64 time)
 
                 if (STATUS_SUCCEEDED(retStatus)) {
                     pSignalingClient->getEndpointTime = time;
+
+                    if (pSignalingClient->pChannelInfo->cachingPolicy == SIGNALING_API_CALL_CACHE_TYPE_FILE) {
+                        signalingFileCacheEntry.creationTsEpochSeconds = GETTIME() / HUNDREDS_OF_NANOS_IN_A_SECOND;
```

Comment:
This was resolved but I still see GETTIME() call. Just use the time variable that's passed in